### PR TITLE
Update Information_storage.rst

### PR DESCRIPTION
**Pull request by satyaog**
Add scratch location in $HOME and full scratch path

### CODE REVIEW

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Comment by satyaog** [comment]
quick look url: https://mila-docs-staging--103.org.readthedocs.build/en/103/Information.html#storage

File: docs/Information_storage.rst [hunk 1]
```diff
@@ -5,22 +5,22 @@ Storage
...
+Path                                                   Performance Usage                                  Quota (Space/Files) Auto-cleanup
+====================================================== =========== ====================================== =================== ============
+``$HOME`` or ``/home/mila/<u>/<username>/``            Low         * Personal user space                  100G/1000K
+                                                                   * Specific libraries, code, binaries
+``/network/projects/<groupname>/``                     Fair        * Shared space to facilitate           200G/1000K
+                                                                     collaboration between researchers
+                                                                   * Long-term project storage
+``/network/data1/``                                    High        * Raw datasets (read only)
+``/network/datasets/``                                 High        * Curated raw datasets (read only)
+``~/scratch/`` or ``/network/scratch/<u>/<username>/`` High        * Temporary job results                                    90 days
```
Comment (satyaog) [inline docs/Information_storage.rst:18]:
```suggestion
``$SCRATCH`` or ``/network/scratch/<u>/<username>/`` High          * Temporary job results                                    90 days
```

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

File: docs/Information_storage.rst (hunk 1 above)
Comment (btravouillon) [reply docs/Information_storage.rst]: I updated the PR with the proposal above: use of `$SCRATCH`.

**Review by ahmam — APPROVED** [review]

**Review by achillemyette1 — APPROVED** [review]

**Pull request merged**